api/block/routes_test.go

```diff
@@ -46,7 +46,7 @@ func startNodeServer(handler blockatlas.FacadeHandler) *gin.Engine {
 	ws.Use(cors.Default())
 	blockRoutes := ws.Group("/block")
 	if handler != nil {
-		blockRoutes.Use(api.WithElrondProxyFacade(handler))
+		blockRoutes.Use(api.WithElrondProxyFacade(handler, "v1.0"))
```

Comment:
Old file, it was removed.  And yes, the tests rely on `v1.0` because other versions handlers are added over the `v1.0` and their tests will be placed there